modules/templates/helper.go

```diff
@@ -795,6 +796,14 @@ func RenderCommitBody(ctx context.Context, msg, urlPrefix string, metas map[stri
 	return template.HTML(renderedMessage)
 }
 
+// Match text that is between back ticks. 
+var codeMatcher = regexp.MustCompile("`([^`]+)`")
+
+func RenderCodeBlock(htmlEscapedTextToRender template.HTML) template.HTML {
```

Comment:
```suggestion
// RenderCodeBlock renders `…` as markdown highlight.
// Intended for issue and PR titles, hence these elements should also have the 'issue-title' class
func RenderCodeBlock(htmlEscapedTextToRender template.HTML) template.HTML {
```

Reply:
`hence these elements should also have the 'issue-title' class`.

Hmm, that's not true at the moment.

Some elements' `issue-title` are put on their parent elements. Some elements already have children `code` styles, so, the comment doesn't cover these cases. I'd like to take the first line comment

Reply:
```suggestion
// RenderCodeBlock renders "`…`" as highlighted "<code>" block.
// Intended for issue and PR titles, these containers should have styles for "<code>" elements
func RenderCodeBlock(htmlEscapedTextToRender template.HTML) template.HTML {
```